tests/unit/makefile

```
@@ -243,15 +271,22 @@ ifeq ($(MFEM_USE_CUDA),YES)
 	@$(call mfem-test,$<,, CEED Unit tests (cuda-gen),--device ceed-cuda:/gpu/cuda/gen $(MFEM_DATA_FLAG),SKIP-NO-VIS)
 endif
 
+mjit_tests-test-seq: mjit_tests
+	@$(call mfem-test,$<,, Mjit tests,--device cpu,SKIP-NO-VIS)
```

I think this should be just

```suggestion
	@$(call mfem-test,$<,, Unit tests,--device cpu,SKIP-NO-VIS)
```

which will show e.g. as 

```
    Unit tests [ sedov_tests_cpu ... ]: OK  (6.73s 149260kB)
    Unit tests [ sedov_tests_debug ... ]: OK  (2.36s 6984kB)
    Unit tests [ tmop_pa_tests_cpu ... ]: OK  (5.13s 148560kB)
    Unit tests [ tmop_pa_tests_debug ... ]: OK  (9.95s 12688kB)
    Unit tests [ mjit_tests ... ]: OK  (24.09s 147224kB)
    Unit tests [ debug_device_tests ... ]: OK  (1.06s 3324kB)
```

---

`done`